docs/src/optimization_packages/nonconvex.md

```diff
@@ -0,0 +1,65 @@
+# Nonconvex.jl
+[`Nonconvex`](https://github.com/JuliaNonconvex/Nonconvex.jl) is a is a Julia package implementing and wrapping nonconvex constrained optimization algorithms.
+
+A `Nonconvex` algorithm is called using one of the following:
```

Comment:
I would recommend referring to the Nonconvex docs. Keeping track of changes here will be difficult for you :)

Reply:
I think this combination is good, a quick summary + a reference.